### PR TITLE
feat : add readme & responsiveness

### PR DESCRIPTION
**Pull request by sudipkundu999**
In this PR, I've changed the landing page, added readme and responsiveness to the pages
COBRA-STORE

### CODE REVIEW

File: pages/cart/cart.css [hunk 1]
```diff
@@ -6,11 +6,12 @@
 }
 
 main{
+    max-width: 1080px;
```
Comment (tonymike99) [inline pages/cart/cart.css:9]: Hey! You can avoid using px.

**Review by tonymike99 — APPROVED** [review]
Hey! Everything looks great! UI is stunning.
Suggestion: You've used px for values above 10. Instead, you can use rem.
Otherwise, everything looks great!

File: homepage.css [hunk 2]
```diff
@@ -6,58 +6,66 @@
...
+    background-color: var(--black);
+}
   
 body{
     background-color: var(--BG-WHITE-LIGHT);
 }
 
 main{
-    padding: 5rem;
     display: flex;
```
Comment (sach10-create) [inline homepage.css:21]: can be clubbed in a class, as it is used often

Comment (sach10-create) [inline homepage.css:21]: can be clubbed in one class, as it is used often

**Review by sach10-create — APPROVED** [review]
Good to merge, common properties can be clubbed together

**Review by GarvG — APPROVED** [review]
looks good to me

File: homepage.css (hunk 2 above)
Comment (sudipkundu999) [reply homepage.css:21]: Thank you for the suggestion,
I will do that when I add utility classes in CL during refactoring.

File: pages/cart/cart.css (hunk 1 above)
Comment (sudipkundu999) [reply pages/cart/cart.css]: Done!

**Pull request merged**